apps/ledger-live-mobile/src/families/cosmos/Delegations/index.tsx

```diff
@@ -70,7 +70,7 @@ function Delegations({ account }: Props) {
   const { t } = useTranslation();
   const mainAccount = getMainAccount(account) as CosmosAccount;
   const delegations: CosmosMappedDelegation[] = useCosmosFamilyMappedDelegations(mainAccount);
-
+  const isCroAccount = account.type === "Account" && account.currency.id === "crypto_org";
```

Comment:
What do you think of adding Cro to your `getAccountType` function and handling it as the other coins ?